tils/drupal-modules.mdx

```diff
@@ -0,0 +1,112 @@
+---
```

Comment:
@NeoMax97 I'd like to keep TILs specific in terms of what you have learned. I feel we are conveying two things here: 1. Different ways to install a module and 2. How to install a module using Composer. I'd suggest splitting it into two posts. This one can be #2.